qubesagent/xdg.py

```diff
@@ -117,6 +117,13 @@ def launch(filename_or_entry, *files, **kwargs):
                     print(e)
                     # fallback to non-dbus version
                     pass
+                except ValueError as e:
+                    if str(e).startswith('Invalid bus name'):
```

Comment:
TBH, I hate this way of checking for the error message, especially since it might be language-dependent. Maybe you can wrap just the `bus.get_object()` call in a (separate) "try/except/else ValueError" instead (and don't look at the message in that case)?

Reply:
Is this workaround also applicable to Qubes R4.1.2, where the issue does exist, too?

Reply:
I think so.